server/service/src/invoice_line/stock_out_line/update/generate.rs

```diff
@@ -120,6 +120,7 @@ fn generate_line(
         r#type,
         note: input.note,
         inventory_adjustment_reason_id: None,
+        return_reason_id: None,
```

Comment:
Following on from earlier - if I have a separate `update_return_reason_id` method, is it fine to always set `return_reason_id` to `None` here, and then update separately, or should I pull `return_reason_id` off the input and pass through here?

Reply:
Yeah i think it's fine to be None here for now, and we always update_return_reason_id for return updates, which as you've mentioned will be done in the update PR, thanks.